src/Tools/dotnet-dump/Analyzer.cs

```diff
@@ -56,7 +56,19 @@ public async Task<int> Analyze(FileInfo dump_path, string[] command)
                     target = DataTarget.LoadCoreDump(dump_path.FullName);
                 }
                 else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
-                    target = DataTarget.LoadCrashDump(dump_path.FullName, CrashDumpReader.ClrMD);
+                    try
+                    {
+                        target = DataTarget.LoadCoreDump(dump_path.FullName);
+                    }
+                    catch (InvalidDataException)
+                    {
+                        // This condition occurs when we try to load a Windows dump as a Elf core dump.
+                    }
+
+                    if (target == null)
+                    {
+                        target = DataTarget.LoadCrashDump(dump_path.FullName, CrashDumpReader.ClrMD);
+                    }
```

Comment:
I debated putting this clause in the catch block...

Reply:
I am leaning to moving inside as I stare at it now...

Reply:
I think it is fine to be outside.  There is already a try/catch around the all of this code.